addons/pos_sale/models/sale_order.py

```diff
@@ -101,12 +101,15 @@ def _compute_qty_invoiced(self):
     def _get_sale_order_fields(self):
         return ["product_id", "display_name", "price_unit", "product_uom_qty", "tax_id", "qty_delivered", "qty_invoiced", "discount", "qty_to_invoice", "price_total", "is_downpayment"]
 
+    def get_product_uom(self):
```

Comment:
- Make it private
- Maybe a more specific name to avoid colliding with other modules, something like `_get_product_uom_pos`

If this  patch is not accepted, you could implement a workaround in Ventacero like:

```
def read_converted(self):
    products_with_conversion = self.product_id.filtered("uom_conversion_id")
    original_uoms = products_with_conversion.read(["uom_id"])
    for product in products_with_conversion:
        product._cache["uom_id"] = product.uom_conversion_id.id
    res = super().read_converted()
    for product, uom in zip(products_with_conversion, original_uoms):
        product._cache["uom_id"] = uom["uom_id"][0]
    return res
```

That way, this patch wouldn't be required.